src/components/Coaching/CoachingList.tsx

```diff
@@ -57,9 +63,9 @@ export const CoachingList: React.FC<CoachingListProps> = ({
       <Box>
         {loading ? (
```

Comment:
This won't render anything until all the pages load. Can we show the skeletons if `!data && !error`?